frontend/packages/noobaa-storage-plugin/src/components/bucket-class/wizard-pages/backingstore-page.tsx

```diff
@@ -26,8 +26,7 @@ const BackingStorePage: React.FC<BackingStorePageProps> = React.memo(
           >
             <p>
               {t(
-                'noobaa-storage-plugin~Backing Store represents a storage target to be used as the underlying storage for the data in {{MCG}} buckets.',
-                { mcg: 'MCG' },
+                'noobaa-storage-plugin~Backing Store represents a storage target to be used as the underlying storage for the data in MCG buckets.',
```

Comment:
```suggestion
                'noobaa-storage-plugin~Backing Store represents a storage target to be used as the underlying storage for the data in MultiCloud Object Gateway buckets.',
```
Please fix all instances of this. Thanks

Reply:
This can be handled in a separate PR.